config/routes.rb

```diff
@@ -10,6 +10,7 @@
   namespace :admin do
     resources :books
     resources :dashboard, only: :index
```

Comment:
I realize this isn't part of this pr, but is dashboard actually a resource? If it's just a page you can just do `get 'dashboard'` and whatever options you need to pass. If it's a singular resource you can use `resource 'dashboard', only: :show`.